lib/glimesh/streams.ex

```diff
@@ -94,14 +95,22 @@ defmodule Glimesh.Streams do
     Repo.get_by(Channel, user_id: user.id) |> Repo.preload([:category, :user])
   end
 
-  def create_channel(user, attrs \\ %{}) do
+  def create_channel(user, attrs \\ %{category_id: 1}) do
```

Comment:
What is category id 1?

Reply:
It's currently the Art category. I had to populate it due to when you created your channel with the "Create Channel" button your stream page would break if you didn't set anything.

Reply:
Changed this to get the list of categories and just pick the first one it finds. It's needed to be set on channel creation otherwise the page will break since category_id would be null.